### DISCUSSION TITLE
Agent Committee Meeting Wednesday 12/08/2021

### DISCUSSION THREAD

**Discussion by lin-fred** [ArctosDB/arctos#6127] [Agent Committee]
Hey everyone,

Reminder the next Membership/Agent meeting is Wednesday at 2pm MTN time. You can see the Arctos calendar for details. Please contact me if you have issues using the zoom information.

Take a look at the agents project page https://github.com/ArctosDB/arctos/projects/11 under the Agent Committee Discussion. Look over the issues, we will be going over them in order from top to bottom.

Please add any issues that any of you feel we need to discuss, or think is something that we could get done with one meeting of discussion. 

See you all then!

**Comment by lin-fred** [comment]
agenda https://docs.google.com/document/d/1CHu_9ZzDl3s-JI1CDLAyXDLNg6Uec4hhX42xWV1TGpY/edit#

**Comment by droberts49** [comment]
I will miss this one -- have to take kiddos to a doctor appt.

**Comment by SerinaBrady** [comment]
Sorry, I won't be able to make this meeting today!

Best,
Serina
________________________________
